creator-node/scripts/start.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+if [[ "$WAIT_HOSTS" != "" ]]; then
+    /wait
```

Comment:
is this the correct path? i thought it was /usr/bin/wait

Reply:
Fixed and tested again